addons/web/static/src/search/filter_menu/advanced_search_dialog.js

```diff
@@ -31,6 +31,7 @@ export class AdvancedSearchDialog extends Component {
             domain,
             onSave: (domain) => this.onSelected(domain),
             resModel,
+            isDebugMode: !!this.env.debug,
```

Comment:
maybe the default should be env.debug. It is sometime set to true. Never to false